### ISSUE TITLE
S4 methods

### ISSUE THREAD

**Issue by mjsteinbaugh** [jennybc/access-r-source#4]
Hi @jennybc, would you be up for merging a pull request that documents how to access S4 methods? If so, I can work on adding that to the markdown.

Best,
Mike

**Comment by jennybc** [comment]
Yeah, sure, if it's of an appropriate "size" (as in, length relative to rest of stuff).

This document is quite static and I think the lookup package is now a great choice for finding source code. But it is also nice to have written out and to understand things better one's self.

**Comment by mjsteinbaugh** [comment]
The lookup package is great, especially for debugging compiled code. I thought it would be handy to append the S4 accessors in the document, since this question comes up a lot with Bioconductor packages.
